src/PAModel/Entropy.cs

```diff
@@ -56,6 +56,11 @@ internal class PropertyEntropy
 
         public PropertyEntropy VolatileProperties { get; set; }
 
+        // Sometimes, an empty LocalDatabaseReferences is serialized as "" or { }.
+        // Need to track which so we can round-trip.
+        // if true: "".   Else,  "{}". 
+        public bool LocalDatabaseReferencesAsEmpty { get; set; }
```

Comment:
Which is the more up-to-date style of serializing this? Is there something we should add to #21 to track this?